backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -148,7 +147,7 @@ def __init__(self, smtp_config: SmtpConfiguration, config: CFG, session: Session
         #     session_factory = get_session_factory(engine)
         #     app_config = CFG(settings)
 
-    def _get_sender(self, user: User = None) -> str:
+    def _get_sender(self, user: User = None) -> EmailAddress:
         """
         Return sender string like "Bob Dylan
```

Comment:
This docstring probably needs to be updated

Reply:
done